devel/libimobiledevice-glue/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        libimobiledevice libimobiledevice-glue 1.0.0
+revision            0
```

Comment:
please add "github.tarball_from" and use either "releases" or "archive", whichever is applicable. You'll need to recalculate the "checksums".

Reply:
Added to updated or new ports. The rest of the ports will have this added in the next update (otherwise it's going to conflict with already mirrored dist files)